system/src/commonMain/kotlin/network/NetworkManager.kt

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright 2022 Splendo Consulting B.V. The Netherlands
+ Copyright 2023 Splendo Consulting B.V. The Netherlands
```

Comment:
Looks like automatically generated by renaming/moving file, but should we 'fix' such changes like `2022-2023`?